cherrypy/test/test_static.py

```diff
@@ -17,6 +20,17 @@
 from cherrypy.test import helper
 
 
+@pytest.fixture
+def unicode_filesystem(tmpdir):
+    filename = tmpdir / ntou('☃', 'utf-8')
+    tmpl = "File system encoding ({encoding}) cannot support unicode filenames"
+    msg = tmpl.format(encoding=sys.getfilesystemencoding())
+    try:
+        io.open(six.text_type(filename), 'w')
```

Comment:
Yeah, I'm slightly tempted not to use io at all and simply test encoding of the filename with `getfilesystemencoding`. As it is, that file handle will never get assigned and will get garbage-collected almost immediately, but you're right that it's not explicitly closed. Perhaps it is worthwhile to close it explicitly. I'll do that.